solc/CommandLineInterface.cpp

```diff
@@ -88,6 +88,16 @@ using namespace solidity;
 using namespace solidity::util;
 using namespace solidity::langutil;
 
+namespace
+{
+
+std::set<frontend::InputMode> ValidInputModes{
```

Comment:
Well, all input modes are valid :) The common thing among these is that they run the compiler (as opposed to assembler, linker, etc.)
```suggestion
std::set<frontend::InputMode> CompilerInputModes{
```

Reply:
True :D